test/jsonValidationTest.js

```diff
@@ -42,6 +42,18 @@ describe('Validating Cohorts...', () => {
             done();
         });
     });
+    
+    it('Verifying cohort category or settings json exists', function (done) {
+        browseDirectory(cohortPath, (error, results) => {
+            if (error) throw error;
+            results.filter(file => file.substr(-9) === '.cohort')
+                .forEach(folder => {
+                    validateJsonExistForWorkbook(cohortPath, results, folder.substr(cohortPath.length+1))
```

Comment:
does this work for cohorts? don't the files end in `.cohort?`  getProgressivePaths only seems to be looking for `.workbook`

Reply:
should filter out cohort